packages/lumberaxe/spec/lumberaxe/lumberaxe_spec.rb

```diff
@@ -3,45 +3,57 @@
-    post "/campgrounds", params: { campground: { name: "Cloudland Canyon" }, format: :json }
-  end
-
-  it "tags request_id" do
-    expect { subject }.to output(/"request_id":/).to_stdout_from_any_process
-  end
-
-  it "tags IP" do
-    expect { subject }.to output(/"IP":/).to_stdout_from_any_process
-  end
-
-  it "contains the defined progname" do
-    expect { subject }.to output(/"progname":"app"/).to_stdout_from_any_process
-  end
-
-  it "contains message key/value pairs" do
-    expect { subject }.to output(/"message":/).to_stdout_from_any_process
-  end
-
-  it "logs HTTP requests" do
-    expect { subject }.to output(%r{"method":"POST","path":"/campgrounds"}).to_stdout_from_any_process
-  end
-
-  it "logs any params" do
-    expect { subject }.to output(/"params":{"campground":{"name":"Cloudland Canyon"}/).to_stdout_from_any_process
-  end
-
-  it "logs anything passed to the rails logger" do
-    expect { subject }.to output(/Creating campground named Cloudland Canyon/).to_stdout_from_any_process
-  end
-
-  it "logs DB requests" do
-    expect { subject }.to output(/INSERT INTO/).to_stdout_from_any_process
-  end
-
-  it "logs error response" do
-    expect do
-      post "/campgrounds", params: { campground: { name: nil } }
-    end.to output(/"status":422/).to_stdout_from_any_process
+  context "with a standard request" do
+    subject do
+      post "/campgrounds", params: { campground: { name: "Cloudland Canyon" }, format: :json }
+    end
+
+    it "tags request_id" do
+      expect { subject }.to output(/"request_id":/).to_stdout_from_any_process
+    end
+
+    it "tags IP" do
+      expect { subject }.to output(/"IP":/).to_stdout_from_any_process
+    end
+
+    it "contains the defined progname" do
+      expect { subject }.to output(/"progname":"app"/).to_stdout_from_any_process
+    end
+
+    it "contains message key/value pairs" do
+      expect { subject }.to output(/"message":/).to_stdout_from_any_process
+    end
+
+    it "logs HTTP requests" do
+      expect { subject }.to output(%r{"method":"POST","path":"/campgrounds"}).to_stdout_from_any_process
+    end
+
+    it "logs any params" do
+      expect { subject }.to output(/"params":{"campground":{"name":"Cloudland Canyon"}/).to_stdout_from_any_process
+    end
+
+    it "logs anything passed to the rails logger" do
+      expect { subject }.to output(/Creating campground named Cloudland Canyon/).to_stdout_from_any_process
+    end
+
+    it "logs DB requests" do
+      expect { subject }.to output(/INSERT INTO/).to_stdout_from_any_process
+    end
+
+    it "logs error response" do
+      expect do
+        post "/campgrounds", params: { campground: { name: nil } }
+      end.to output(/"status":422/).to_stdout_from_any_process
+    end
+  end
+
+  context "#puma_formatter" do
+    subject do
+      Lumberaxe.puma_formatter.call("test log message")
+    end
+
+    it "returns log message" do
+      expect(subject).to include("test log message")
+    end
```

Comment:
this is the meat of the spec coverage I added; it doesn't really reproduce the problem (the other specs will ensure rails gets loaded and I don't know of a way to isolate these like we'd need), but if there is a way, please let me know.

Reply:
What if you create another spec, say `puma_formatter_spec`, and load spec helper instead?